services/cd-service/pkg/service/git.go

```diff
@@ -24,10 +24,15 @@ import (
 	"strings"
 
 	api "github.com/freiheit-com/kuberpult/pkg/api/v1"
-	"github.com/freiheit-com/kuberpult/pkg/valid"
 	grpcErrors "github.com/freiheit-com/kuberpult/pkg/grpc"
+	"github.com/freiheit-com/kuberpult/pkg/logger"
+	"github.com/freiheit-com/kuberpult/pkg/uuid"
+	"github.com/freiheit-com/kuberpult/pkg/valid"
+	eventmod "github.com/freiheit-com/kuberpult/services/cd-service/pkg/event"
 	"github.com/freiheit-com/kuberpult/services/cd-service/pkg/repository"
+	billy "github.com/go-git/go-billy/v5"
```

Comment:
Have you checked if `io/fs` also fulfills your needs for this? Would be more standard and save a dependency.

Reply:
good question, I suspect it would work too. However, we do use billy everywhere, so it would be odd to change it just here. We should discuss this in a refinement.

Reply:
Sure, if you're already using it, then it's fine for now.